examples/multiple_streams_one_portal.py

```diff
@@ -0,0 +1,45 @@
+import trio
```

Comment:
This is just an example that shows the hang in practice.

The problem is if we have a nursery exit before receiving a ctr-c we'll be stuck in the shielded `proc.wait()` and have no way for the user to trigger graceful exit...